dataplicity/m2mmanager.py

```diff
@@ -119,12 +119,18 @@ def run(self):
         while 1:
             # Get the identity, and tell the server about it
             identity = self._identity = self.m2m_client.wait_ready(20)
+            log.debug('%0.1fs since last packet', self.m2m_client.time_since_last_packet)
```

Comment:
Will keep this logging in for now. It's excessively verbose, but it won't render in the user's logs.